distributed/_version.py

```diff
@@ -84,7 +84,7 @@ def run_command(commands, args, cwd=None, verbose=False, hide_stderr=False, env=
                 stderr=(subprocess.PIPE if hide_stderr else None),
             )
             break
-        except EnvironmentError:
+        except OSError:
             e = sys.exc_info()[1]
             if e.errno == errno.ENOENT:
```

Comment:
Maybe this could be simplified like so?

```suggestion
        except OSError as e:
            if e.errno == errno.ENOENT:
```

Reply:
definitely but I didn't want to actually change things in here manually. This PR is merely for the automation part